arangod/AsyncRegistryServer/Feature.h

```diff
@@ -43,10 +43,12 @@ class Feature final : public application_features::ApplicationFeature {
                                                  name()},
         metrics{create_metrics(
             server.template getFeature<arangodb::metrics::MetricsFeature>())} {
-    coroutine_registry.set_metrics(metrics);
+    registry.set_metrics(metrics);
     // startsAfter<Bla, Server>();
   }
 
+  ~Feature() { registry.set_metrics(std::make_shared<Metrics>()); }
```

Comment:
Why do we add new metrics to the registry in dtor?

Reply:
We replace it with an default constructed version of the metrics.

Reply:
I'm also not entirely sure if we need this 🤔